.github/workflows/release.yml

```diff
@@ -135,6 +136,7 @@ jobs:
           git commit -m "Update to version ${{ github.ref_name }}"
           git show
           git push git@github.com:cockpit-project/org.cockpit_project.CockpitClient HEAD
+          gh auth login
```

Comment:
Note to self: https://cli.github.com/manual/gh_auth_login documents that setting `GH_TOKEN` is the right thing to do.